File: features/transaction/purchase_platform/src/main/java/com/tokopedia/purchase_platform/features/cart/data/model/response/ProductShowCase.java
package com.tokopedia.purchase_platform.features.cart.data.model.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author anggaprasetiyo on 31/01/18.
 */

public class ProductShowCase {

    @SerializedName("name")
    @Expose
    private String name = "";
    @SerializedName("id")
    @Expose
    private int id;

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }
}

File: features/transaction/purchase_platform/src/main/java/com/tokopedia/purchase_platform/features/cart/data/model/response/shopgroupsimplified/ProductShowCase.kt
package com.tokopedia.purchase_platform.features.cart.data.model.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author anggaprasetiyo on 31/01/18.
 */

data class ProductShowCase(
    @SerializedName("name")
    @Expose
    val name: String = "",
    @SerializedName("id")
    @Expose
    val id: Int = 0
)
